src/client/components/note-header.vue

```diff
@@ -12,7 +12,10 @@
 		<router-link class="created-at" :to="note | notePage">
 			<mk-time :time="note.createdAt"/>
 		</router-link>
-		<span class="visibility" v-if="note.visibility != 'public'">
+		<span class="visibility">
+			<fa v-if="note.localOnly" :icon="faBiohazard"/>
+			<template v-if="note.localOnly">&nbsp;</template>
```

Comment:
This is a matter of preference, but I think putting <fa> in the <template> would be beautiful because it would have one v-if.

example:

```vue
			<template v-if="note.localOnly">
				<fa :icon="faBiohazard"/>
				&nbsp;
			</template>
```